tests/meta/rsync/test_rsync.py

```diff
@@ -181,6 +169,44 @@ def test_concatenate_rsync_commands(
     )
 
 
+def test_concatenate_rsync_commands_mutant(
+    rsync_api: RsyncAPI,
+    folders_to_deliver: set[Path],
+    analysis_family: dict,
+    analysis_store_trio,
+    project_dir,
+    customer_id,
+    ticket_id: str,
+    case: Case,
+    mocker,
+):
+    """Tests the function to concatenate rsync commands for transferring multiple files."""
+    # GIVEN a list with a mutant case and a sample name and a mutant report file
+    case.data_analysis = Workflow.MUTANT
+    source_and_destination_paths = {
+        "delivery_source_path": project_dir / customer_id / ticket_id,
+        "rsync_destination_path": project_dir / customer_id,
+    }
+    report_path = Path(project_dir, customer_id, ticket_id, "mutant_report.txt")
```

Comment:
Use FileExtension

Reply:
the file name and extension does not really matter here. I changed it to a name without extension